File: 이승현/16503 괄호 없는 사칙연산.c
#include <stdio.h>

int calculator(int a, int b, int op)
{
	int ans;
	switch (op)
	{
	case '+':
		ans = a + b;
		break;
	case '-':
		ans = a - b;
		break;
	case '*':
		ans = a * b;
		break;
	case '/':
		ans = a / b;
		break;
	}
	return ans;
}
int main()
{
	int k1, k2, k3;
	char o1, o2;
	int ans1 = 0, ans2 = 0;
	scanf("%d %c %d %c %d", &k1, &o1, &k2, &o2, &k3);
	ans1 = calculator(k1, k2, o1);
	ans1 = calculator(ans1, k3, o2);
	ans2 = calculator(k2, k3, o2);
	ans2 = calculator(k1, ans2, o1);
	if (ans1 < ans2)
		printf("%d\n%d", ans1, ans2);
	else
		printf("%d\n%d", ans2, ans1);
	return 0;
}

File: 김예경/16503 괄호 없는 사칙연산.py
#입력받기
k1, o1, k2, o2, k3 = input().split()
k1, k2, k3 = int(k1), int(k2), int(k3)

#수를 입력해 계산
def calculate(a, op, b):
    if op == '+':
        ans = a + b
    elif op == '-':
        ans = a - b
    elif op == '*':
        ans = a * b
    elif op == '/':
        ans = a // b
        if a<0 or b<0:
            ans = ans+1
    return ans

# 앞부터 연산
temp = calculate(k1, o1, k2)
ans1 = calculate(temp, o2, k3)
# 뒤부터 연산
temp = calculate(k2, o2, k3)
ans2 = calculate(k1, o1, temp)

# 출력하기
if ans1 < ans2:
    print(ans1)
    print(ans2)
else:
    print(ans2)
    print(ans1)